doc/man3/OPENSSL_LH_COMPFUNC.pod

```diff
@@ -29,6 +30,11 @@ OPENSSL_LH_get_down_load, OPENSSL_LH_set_down_load, OPENSSL_LH_error
  LHASH_OF(TYPE) *lh_TYPE_new(OPENSSL_LH_HASHFUNC hash, OPENSSL_LH_COMPFUNC compare);
  void lh_TYPE_free(LHASH_OF(TYPE) *table);
  void lh_TYPE_flush(LHASH_OF(TYPE) *table);
+ OPENSSL_LHASH *OPENSSL_LH_set_thunks(OPENSSL_LHASH *lh,
+                                      OPENSSL_LH_HASHFUNCTHUNK hw,
+                                      OPENSSL_LH_COMPFUNCTHUNK cw,
+                                      OPENSSL_LH_DOALL_FUNC_THUNK daw,
+                                      OPENSSL_LH_DOALL_FUNCARG_THUNK daaw)
```

Comment:
Actual documentation seems to be missing.

Reply:
I had a concern regarding this, and I'm not sure how to handle it.  The thunking functions need to be public here, based on how the DEFINE_LHASH_OF macro is constructed, but users should never have to call it directly.  As such, what should the documentation say here?  something along the lines of "These functions exist for ubsan compatibility, and they should not be called directly"?  Or is there some way to excuse public functions from documentation?

Reply:
added docs to mark the offending functions as actually being internal, and that they should not be used